src/app/pages/enter-reset-password/enter-reset-password.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+<div class="row mt-5">
+        <div class="col-md-3"></div>
+        <div class="col-md-6" >
+
+    <img class="img-fluid align-middle center"  src="../../../assets/img/Logo.png" alt="landville-logo" width="150" height="95"/>
+    <h1 class="text-center">Reset your password</h1>
+    <p class="text-center">Please choose a new password to sign in.</p>
+    <span *ngIf="successMessage" class="text-success">Your password has been reset. </span> 
+    <span *ngIf="tokenError" class="text-danger">Your session has expired, Please restart the process.</span> 
+    <form [formGroup]="enterPasswordForm" (ngSubmit)="onSubmit()">
+
+        <div class="form-group" [ngClass]="{'has-error': enterPasswordForm.get('newPassword').errors &&
+        (enterPasswordForm.get('newPassword').touched || enterPasswordForm.get('newPassword').dirty)}">
+        <input formControlName="newPassword" placeholder="Enter new password"
+                class="form-control" type="password">
+        <span class="help-block" *ngIf="enterPasswordForm.get('newPassword').errors &&
+        (enterPasswordForm.get('newPassword').touched || enterPasswordForm.get('newPassword').dirty)">
+            <span class="text-danger" *ngIf="enterPasswordForm.get('newPassword').errors && enterPasswordForm.get('newPassword').errors.required">
+                Password is required
+        </span>
+        <span class="text-danger" *ngIf="enterPasswordForm.get('newPassword').errors && enterPasswordForm.get('newPassword').errors.minlength">
+                Password should be at least 6 characters
+        </span>
+    </span><br/>
+        </div>
+        <div class="form-group" [ngClass]="{'has-error': passwordError}">
+        <input formControlName="confirmPassword" placeholder="Confirm new password" 
```

Comment:
Hey @joelethan, I don't tend to see validations for the confirmed password. What if someone inputs a none corresponding password. Do you first call the back end to check or

Reply:
No, I do this check from the frontend before hitting the backend. When the passwords are different, the buttons are actually disabled.